src/utah-design-system/Spinner.jsx

```diff
@@ -0,0 +1,34 @@
+import clsx from 'clsx';
+import PropTypes from 'prop-types';
+
+export default function Spinner({ className }) {
```

Comment:
**todo**: this seems to be missing all of the screen reader aria and role information.

https://designsystem.dev.utah.gov/library/components/widgetsIndicators/spinners

Reply:
This gets tricky very quickly. Some of those props are intended to be put on containers. I'll add the few that I think might work for now.

Reply:
dc853d8a1ccc834d1f7301a6da873e0a9f988e4f